### PR TITLE
chore: bump nx

### PR DESCRIPTION
**Pull request by alszczep**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,3 +39,5 @@ testem.log
 Thumbs.db
 
 lerna-debug.log
+
+.nx/cache
```
Comment (Komoszek) [inline .gitignore:43]: newline

File: .prettierignore [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 /dist
 /coverage
+
+/.nx/cache
```
Comment (Komoszek) [inline .prettierignore:6]: newline

File: packages/api-admin/src/lib/components/Table/filters/rangeFilters/NumberRangePickerFilter/index.tsx [hunk 3]
```diff
@@ -19,14 +19,14 @@ export function NumberRangePickerFilter({
         <InputNumber
           placeholder="From"
           size="small"
-          value={selectedKeys[0]}
+          value={typeof selectedKeys[0] === "bigint" ? selectedKeys[0].toString() : selectedKeys[0]}
```
Comment (Komoszek) [inline packages/api-admin/src/lib/components/Table/filters/rangeFilters/NumberRangePickerFilter/index.tsx:22]: Extract to some function?

**Review by Komoszek — APPROVED** [review]

File: packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ export function NumberPickerFilter({ setSelectedKeys, selectedKeys, confirm, cle
       <InputNumber
         placeholder="Input number"
         size="small"
-        value={selectedKeys[0]}
+        value={typeof selectedKeys[0] === "bigint" ? selectedKeys[0].toString() : selectedKeys[0]}
```
Comment (katarzynagrasewicz) [inline packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx:17]: Shouldn't `bigIntToString` be used here as well?

Comment (katarzynagrasewicz) [reply packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx:17]: Also does typeof work in runtime?

Comment (Komoszek) [reply packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx:17]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof :see_no_evil:

Comment (alszczep) [reply packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx]: I have no idea how I overlooked that :claun:

**Review by katarzynagrasewicz — APPROVED** [review]

File: packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx (hunk 4 above)
Comment (katarzynagrasewicz) [reply packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx]: Messed it up with typescript typechecking 🤦

**Pull request merged**